src/libraries/sendtohelixhelp.proj

```diff
@@ -169,7 +169,7 @@
 
       <_WorkItem Include="$(WorkItemArchiveWildCard)" Exclude="$(HelixCorrelationPayload)" />
 
-      <HelixWorkItem Include="@(_WorkItem -> '%(FileName)')">
+      <HelixWorkItem Include="@(_WorkItem -> '%(FileName)')" Condition="'@(_WorkItem)' != ''">
```

Comment:
This will fail on a later step I believe saying that there were no items provided.